src/viam/media/utils.py

```diff
@@ -0,0 +1,74 @@
+from array import array
+from io import BytesIO
+from typing import List
+
+from PIL import Image
+
+from viam.errors import NotSupportedError
+from viam.media.video import CameraMimeType, ViamImage
+
+from .viam_rgba_plugin import RGBA_FORMAT_LABEL
+
+# Formats that are supported by PIL
+LIBRARY_SUPPORTED_FORMATS = ["JPEG", "PNG", RGBA_FORMAT_LABEL]
+
+
+def viam_to_pil_image(image: ViamImage) -> Image.Image:
+    """
+    Convert a ViamImage to a PIL.Image.
+
+    Args:
+        image (ViamImage): The image to convert.
+
+    Returns:
+        Image.Image: The resulting PIL.Image
+    """
+    return Image.open(BytesIO(image.data), formats=LIBRARY_SUPPORTED_FORMATS)
+
+
+def bytes_to_depth_array(image: ViamImage) -> List[List[int]]:
+    """
+    Decode the data of an image that has the custom depth MIME type ``image/vnd.viam.dep`` into a standard representation.
+
+    Args:
+        image (ViamImage): The image to be decoded.
+
+    Raises:
+        NotSupportedError: Raised if given an image that is not of MIME type `image/vnd.viam.dep`.
+
+    Returns:
+        List[List[int]]: The standard representation of the image.
+    """
+    if image.mime_type != CameraMimeType.VIAM_RAW_DEPTH.value:
+        raise NotSupportedError("Type must be `image/vnd.viam.dep` to use bytes_to_depth_array()")
+
+    width = int.from_bytes(image.data[8:16], "big")
+    height = int.from_bytes(image.data[16:24], "big")
+    image.width = width
+    image.height = height
+    depth_arr = array("H", image.data[24:])
+    depth_arr.byteswap()
+
+    depth_arr_2d = [[depth_arr[row * width + col] for col in range(width)] for row in range(height)]
+    return depth_arr_2d
+
+
+def get_image_dimensions(image: ViamImage) -> None:
```

Comment:
(minor) When I see a method named `get_foo` I typically expect that it will return a `foo`, I think it's a bit strange that this returns `None`. But it'd be weird to name it `set_image_dimensions` too because it's taking no `image_dimension` arguments! Can we rename to `determine_image_dimensions` or something like that?

Reply:
Yes, this is a very good point!